nca/CoreDS/TcpLikeProperties.py

```diff
@@ -145,10 +154,10 @@ def get_cube_dict(self, cube, is_txt=False):
             dim_domain = DimensionsManager().get_dimension_domain_by_name(dim)
             if dim_domain == dim_values:
                 continue  # skip dimensions with all values allowed in a cube
-            if dim == 'methods':
+            if dim == 'protocols' or dim == 'methods':
                 values_list = str(dim_values)
-            elif dim == "peers":
-                values_list = self.base_peer_set.get_peer_list_by_indices(dim_values)
+            elif dim == "src_peers" or dim == "dst_peers":
+                values_list = self.base_peer_set.get_peer_set_by_indices(dim_values)
```

Comment:
shouldn't `values_list` be a list of `str` values?

Reply:
It was actually a list of Peers, and I changed it to PeerSet. It does not matter, it is further converted to str.

Reply:
I think that for consistency considerations it may be better to change to list of `str` values as well.

Reply:
Changed